x-pack/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -525,6 +525,9 @@ function buildSuggestion({
     xTitle: currentState?.xTitle,
     yTitle: currentState?.yTitle,
     yRightTitle: currentState?.yRightTitle,
+    hideEndzones: currentState?.hideEndzones,
```

Comment:
This is a bug fix unrelated to the rest of the PR - the hide endzones parameter was reset when switching between xy chart types